src/system/hot.c

```diff
@@ -78,3 +86,11 @@ void invoke_install_hot_table() {
 		memset(HOT_TABLE, 0, sizeof(*HOT_TABLE));
 	}
 }
+
+// This is a callback function used by newlib to get the unix timestamp
+// newlib cannot access any symbols in the hot package, so we have the hot
+// package pass a function pointer to this function. Newlib then uses that
+// function pointer. 
+static const int get_timestamp_int(void) {
+	return _PROS_COMPILE_TIMESTAMP_INT;
+}
```

Comment:
newline at end of file required
```suggestion
}

```